integration-tests/bats/system-tables.bats

```diff
@@ -174,52 +174,61 @@ teardown() {
 }
 
 @test "system-tables: insert into dolt_remotes system table" {
-    skip "Remotes table not yet mutable in SQL session"
     run dolt sql -q "insert into dolt_remotes (name, url) values ('origin', 'file://remote')"
     [ $status -ne 0 ]
     [[ ! "$output" =~ panic ]] || false
 
     mkdir remote
-    dolt sql -q "insert into dolt_remotes (name, url) values ('origin1', 'file://remote')"
-    dolt sql -q "insert into dolt_remotes (name, url) values ('origin2', 'aws://[dynamo_db_table:s3_bucket]/repo_name')"
+    dolt sql -q "CALL DOLT_REMOTE('add', 'origin1', 'file://remote/')"
```

Comment:
Looks like we needed to end with a trailing slash to get this test working? If so, that's a sharp edge that's worth looking into fixing so customers don't have to remember to always end remotes in slashes.

Reply:
the test works with or without trailing slash :)